Frontend/library/src/WebRtcPlayer/WebRtcPlayerController.ts

```diff
@@ -340,6 +340,14 @@ export class WebRtcPlayerController {
         };
     }
 
+    /**
+     * Destroys the player and makes sure resources are freed. This helps to prevent the issue in chrome
+     * where it refuses to make new video players.
+     */
+    destroyPlayer() {
+        this.videoPlayer.destroy();
+    }
+
```

Comment:
```suggestion
    /**
     * Destroys the video player and makes sure resources are freed. This helps to prevent the issue in chrome
     * where it refuses to make new video players.
     */
    destroyVideoPlayer() {
        this.videoPlayer.destroy();
    }

```

I propose using the naming video player because player is a bit ambiguous in the Pixel Streaming context.